sormas-api/src/test/java/de/symeda/sormas/api/utils/DefaultUserHelperTest.java

```diff
@@ -0,0 +1,153 @@
+    private static final String DEFAUTL_NAT_CLIN_USER_PASS = "NatClin";
+    private static final String DEFAUTL_SURV_OFF_USER_PASS = "SurvOff";
+    private static final String DEFAUTL_HOSP_INF_USER_PASS = "HospInf";
+    private static final String DEFAUTL_POE_INF_USER_PASS = "PoeInf";
+
+    @Test
+    public void isDefaultUser() {
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAULT_ADMIN_USERNAME));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_SURV_SUP_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_CASE_SUP_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_CONT_SUP_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_POE_SUP_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_LAB_OFF_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_EVE_OFF_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_NAT_USER_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_NAT_CLIN_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_SURV_OFF_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_HOSP_INF_USER_PASS));
+        assertTrue(DefaultUserHelper.isDefaultUser(DEFAUTL_POE_INF_USER_PASS));
+    }
+
+    @Test
+    public void getDefaultPassword() {
+        assertEquals(DEFAULT_ADMIN_PASS, DefaultUserHelper.getDefaultPassword(DEFAULT_ADMIN_USERNAME).orElse(""));
+        assertEquals(DEFAUTL_SURV_SUP_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_SURV_SUP_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_CASE_SUP_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_CASE_SUP_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_CONT_SUP_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_CONT_SUP_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_POE_SUP_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_POE_SUP_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_LAB_OFF_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_LAB_OFF_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_EVE_OFF_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_EVE_OFF_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_NAT_USER_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_NAT_USER_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_NAT_CLIN_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_NAT_CLIN_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_SURV_OFF_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_SURV_OFF_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_HOSP_INF_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_HOSP_INF_USER_PASS).orElse(""));
+        assertEquals(DEFAUTL_POE_INF_USER_PASS, DefaultUserHelper.getDefaultPassword(DEFAUTL_POE_INF_USER_PASS).orElse(""));
+    }
+
+    @Test
+    public void usesDefaultPassword() {
```

Comment:
Maybe extract a helper-method to check and avoid too much code duplication? Sth like

```
testUsesDefaultPasswordHelper(String username, String defaultPassword) {
  String seed = UUID.randomUUID().toString();
        String randomPass = UUID.randomUUID().toString();
        assertTrue(DefaultUserHelper.usesDefaultPassword(username, PasswordHelper.encodePassword(defaultPassword, seed), seed));
        assertFalse(DefaultUserHelper.usesDefaultPassword(username, PasswordHelper.encodePassword(randomPass, seed), seed));
}
```

And then we could use a loop inside the test instead. wdyt?

Reply:
wdyt @schwzr @Snorzze

Reply:
Yep, already on my list ;)